glusterd2/volgen2/store-utils.go

```diff
@@ -29,8 +30,22 @@ func GetVolfiles() ([]string, error) {
 	volfiles := make([]string, len(resp.Kvs))
 
 	for i, kv := range resp.Kvs {
-		volfiles[i] = string(kv.Key)
+		volFile := strings.TrimPrefix(string(kv.Key), "volfiles/")
+		volfiles[i] = volFile
 	}
 
 	return volfiles, nil
 }
+
+//GetVolfile return perticular volfile info
+func GetVolfile(volfileID string) ([]byte, error) {
```

Comment:
To keep the functions consistent, this should return `string` just like `GetVolfiles()`

Reply:
returning the content as []byte to as volfile content is in string format in ETCD (for better human-readability).